designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/RadioButton.kt

```diff
@@ -108,12 +105,9 @@ fun RadioButton(
 fun RadioButtonBlock(
     orientation: Orientation,
     content: List<RadioButtonData>,
-    itemSelected: RadioButtonData,
-    onItemChange: (RadioButtonData) -> Unit,
+    itemSelected: RadioButtonData?,
+    onItemChange: ((RadioButtonData) -> Unit)? = null,
```

Comment:
why do you consider that the callback can be nullable?. I'm trying to think in case where we don't need callback for the user selection, but I'm not sure

Reply:
@andresmr For the disabled state, we don't need the onItemChange as the component is not selectable.

Reply:
I think the disable is a configuration status but in the code we will always set the callback, I would set as non null

Reply:
I think the disable is a configuration status but in the code we will always set the callback, I would set as non null

Reply:
@andresmr I have make the callback non-nullable.